Percy/utils/ObjectExtensions.cs

```diff
@@ -0,0 +1,17 @@
+namespace PercyIO.Appium
+{
+  public static class ObjectExtensions
+  {
+    public static bool IsTrue(this object obj)
+    {
+      string? str = obj?.ToString()?.ToLower();
+      return str == "true";
+    }
+
+    public static bool IsFalse(this object obj)
+    {
+      string? str = obj?.ToString()?.ToLower();
```

Comment:
```suggestion
      String? str = obj?.ToString()?.ToLower();
```
conistently use `string` keyword or `String` class, dont mix it

Reply:
done. It gets imported from system class so when this is not imported the autocomplete rename it to small.